app/maintenance.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html>
```

Comment:
```suggestion
<html lang="en">
```

According to SonarCloud analysis here should be a lang attribute. 
And I found a nice article about it https://www.w3.org/International/questions/qa-lang-why

What do you think?